invofi/apps/frontend/src/lib/migrations/001_lender_preferences.sql

```
@@ -0,0 +1,60 @@
+  created_at    timestamptz not null default now(),
+  updated_at    timestamptz not null default now()
+);
+
+-- Enforce one row per lender
+create unique index if not exists lender_preferences_lender_id_idx
+  on lender_preferences (lender_id);
+
+-- RLS: each lender can only read / write their own preferences
+alter table lender_preferences enable row level security;
+
+create policy "Lender can manage own preferences"
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)